### ISSUE TITLE
BSS' logged in response: user_id and user_role required

### ISSUE THREAD

**Issue by srodriguezOPT** [sonata-nfv/son-gkeeper#689]
It's needed to have in the BSS the user role (customer/developer) and the user_id. 
@jbonnet , Can you provide them in the login response?

Something like:

{
 "token": {"access_token": "eyJhbGci[...]R2gclGyo"}, 
 "user_id": "123[...]456", 
 "user_role": "customer/developer”
}

**Comment by jbonnet** [comment]
PR #696 improves data so that we can solve the issue, but doesn't cose it

**Comment by jbonnet** [comment]
The GK needs to (from @dang03):
```
GET url = URI("http://<address>:<port>/api/v1/users?username=user01")
from the response, you can obtain it:
 [ { "id": "5e1ef7d8-b5cd-4835-8bcb-68b182140fa9", "createdTimestamp": 1492094414797, "username": "user01", "enabled": true, "totp": false, "emailVerified": false, "firstName": "User", "lastName": "Sample", "email": "user.sample@email.com", "attributes": { "userType": [ "developer" ] }, "disableableCredentialTypes": [ "password" ], "requiredActions": [] } ]
e.g.: "attributes": { "userType": [ "developer" ] }

**Comment by jbonnet** [comment]
Partly closed by PR #698 (user ID still missing)

**Comment by jbonnet** [comment]
@srodriguezOPT 
If the BSS doesn't need the `user_id` then this issue has been closed by PR #698. Please test and let me know...
